docs/src/_guide/transformation.md

```diff
@@ -5,14 +5,9 @@ layout: doc
 weight: 5
 ---
 
-<div class="notification is-info">
-  <p>Using Meltano with Snowflake? Try the new <a href="/guide/transformation#adapter-specific-dbt-transformation-preview">adapter-specific dbt plugin (preview)</a>!</p>
-  <p>If you are interested in an adapter other than Snowflake, continue with <a href="/guide/transformation#dbt-installation-and-configuration-classic"> the classic installation guide </a>.</p>
-</div>
-
 Transformations in Meltano are implemented using dbt. All Meltano generated projects have a `transform/` directory, which is populated with the required configuration, models, packages, etc in order to run the transformations. A transform in Meltano is simply a set of dbt models that can be installed as a package. See the [transform plugin](/concepts/plugins#transforms) docs for more details.
 
-## Adapter-Specific dbt Transformation (Preview)
+## Adapter-Specific dbt Transformation
 
 In alignment with the [dbt documentation](https://docs.getdbt.com/docs/available-adapters), we are working towards supporting adapter-specific installations of `dbt`.
```

Comment:
```suggestion
In alignment with the [dbt documentation](https://docs.getdbt.com/docs/available-adapters), we support adapter-specific installations of `dbt`.
```

Reply:
It won't let me comment on the line below but I'd like to add in language and a link to the Hub to get users to contribute their own!